core/src/types/ethereum.rs

```diff
@@ -0,0 +1,94 @@
+//! Types to do with Ethereum.
+
+use std::fmt;
+use std::num::NonZeroU64;
+use std::ops::{Add, AddAssign, Deref};
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use num256::Uint256;
```

Comment:
couldn't we simply use the uin256 type from `ethabi`? `num256` could be purged from our deps

Reply:
ah, that's a shame